ep/lh/ep-lh-0020.md

```diff
@@ -20,35 +20,49 @@ Each developer have a lot of projects, and everyone can have special settings. T
 
 Each project must have configuration file, called `.linterhub.json`. This file in JSON format which should describe:
 
-- `engines`, `ingores` -  It is a required set of attributes. It allows to understand how to run project with user settings.
+- `engines`, `ignores` -  It is a required set of attributes. `engines` it's array with customs settings of each engine. `ignores` it's array with rules of ingores for run the engine.
 
 ### JSON
 
 The configuration file must have the following properties:
 
 | Prorepty      | Type     | Required | Description |
 | -             | :-:      | :-:      | -           |
-| engines       | object[] | +        | How to run each engine in project |
-| ignores       | object[] | +        | Rules of ignore for run the project |
+| engines       | object[] | +        | Custom settings of engine |
+| ignores       | object[] | +        | Rules of ignore for run the engine |
 
-Each properties for running `engine` must have next properties:
+#### Engines
+
+Each configuration `engine` must have next properties:
 
 | Prorepty      | Type     | Required | Description |
 | -             | :-:      | :-:      | -           |
 | name          | string   | +        | Name of engine |
-| activated     | bool     | -        | Activated or deactivated engine for run in project. Default: `false` |
-| install       | enum     | -        | How to run engine. Values: `local`, `global`, `container`. Default: `local`|
+| enable        | bool     | -        | `enable` or `disable` engine for run in project. Default: `false` |
```

Comment:
Then this property looks like required because in most cases if the engine is added to config it means that it's 'enabled'.